site/static/images/func.svg

```diff
@@ -0,0 +1,749 @@
+entity input--><text fill="#2196F3" font-family="Verdana" font-size="12.5" lengthAdjust="spacing" textLength="70.8333" x="6.25" y="19.8792">input items</text><!--MD5=[c661d977cad040094ee01910682146de]
+entity output--><text fill="#2196F3" font-family="Verdana" font-size="12.5" lengthAdjust="spacing" textLength="80.2083" x="10.9375" y="259.4625">output items</text><!--MD5=[c7ab6722dbd12818cf3935a01d245420]
+entity functionConfig--><text fill="#2196F3" font-family="Verdana" font-size="12.5" lengthAdjust="spacing" textLength="89.5833" x="155.2083" y="19.8792">functionConfig</text><!--MD5=[b5cce2cdbd787adcd8934bb88f8254b3]
+entity results--><text fill="#2196F3" font-family="Verdana" font-size="12.5" lengthAdjust="spacing" textLength="41.6667" x="169.7917" y="259.4625">results</text><!--MD5=[54ca21aa7b5e97c0a74b54f1aab99281]
+link input to FUNC--><path d="M60.4271,44.062 C72.0625,61.3641 87.0521,83.6453 99.1875,101.687 " fill="none" id="input-to-FUNC" style="stroke:#2196F3;stroke-width:3.125;"/><polygon fill="#2196F3" points="102.2083,106.1766,100.4348,96.0718,99.3021,101.8545,93.5194,100.7218,102.2083,106.1766" style="stroke:#2196F3;stroke-width:3.125;"/><!--MD5=[6f660645f44eadd895eb0ae999a8844a]
+link functionConfig to FUNC--><path d="M181.2396,44.062 C169.6042,61.3641 154.6146,83.6453 142.4792,101.687 " fill="none" id="functionConfig-to-FUNC" style="stroke:#2196F3;stroke-width:3.125;stroke-dasharray:7.0,7.0;"/><polygon fill="#2196F3" points="139.4583,106.1766,148.1472,100.7218,142.3646,101.8545,141.2319,96.0718,139.4583,106.1766" style="stroke:#2196F3;stroke-width:3.125;"/><!--MD5=[60d87892e5c7e72320090432dc88c7a6]
+link FUNC to output--><path d="M104.2917,163.8537 C94.0417,181.1558 80.8229,203.437 70.125,221.4787 " fill="none" id="FUNC-to-output" style="stroke:#2196F3;stroke-width:3.125;"/><polygon fill="#2196F3" points="67.4583,225.9683,75.8169,220.0196,70.1095,221.4852,68.6439,215.7778,67.4583,225.9683" style="stroke:#2196F3;stroke-width:3.125;"/><!--MD5=[d1fd4539137c181de4ea06ff81e3f341]
+link FUNC to results--><path d="M137.375,163.8537 C147.625,181.1558 160.8438,203.437 171.5417,221.4787 " fill="none" id="FUNC-to-results" style="stroke:#2196F3;stroke-width:3.125;stroke-dasharray:7.0,7.0;"/><polygon fill="#2196F3" points="174.2083,225.9683,173.0228,215.7778,171.5572,221.4852,165.8498,220.0196,174.2083,225.9683" style="stroke:#2196F3;stroke-width:3.125;"/><!--MD5=[237553599fd46c2601c78757909bc453]
+@startuml
+
+!include ../common/all.puml
+
+rectangle "func" as FUNC
+label "input items" as input
+label "output items" as output
+label functionConfig
+label results
+
+input - -> FUNC
+functionConfig ..> FUNC
+FUNC - -> output
+FUNC ..> results
+
+
+@enduml
+
+@startuml
+
+
+
+
+
+
+skinparam backgroundColor transparent
+
+
+
+
+
+
+
+skinparam defaultFontName       "Verdana"

```

Comment:
Nit: On kpt.dev we use these as the font definition.

```css
--base-font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol"
```

For code blocks we use:

```css
--code-font-family: Inconsolata, Consolas, Menlo, Monaco, "Andale Mono WT", "Andale Mono", "Lucida Console", "DejaVu Sans Mono", "Bitstream Vera Sans Mono", "Courier New", Courier, monospace;
```

It may help to be consistent with that style in these graphics.

Reply:
Filed #1714